packages/entrykit/src/createWagmiConfig.ts

```diff
@@ -21,14 +23,45 @@ export function createWagmiConfig<
   const chains extends readonly [Chain, ...Chain[]],
   transports extends Record<chains[number]["id"], Transport>,
 >(config: CreateWagmiConfigOptions<chains, transports>): Config<chains, transports> {
-  // TODO: wallets so we can extend with eventual passkey wallet
+  // TODO: remove connectors and use ConnectKit's default once https://github.com/wevm/wagmi/pull/4691 lands
+  const connectors: CreateConnectorFn[] = [];
```

Comment:
this pattern of building up connectors array comes from https://github.com/family/connectkit/blob/main/packages/connectkit/src/defaultConnectors.ts
but replaces the walletconnect one with our own